nonce-client/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+name = "solana-nonce-client"
```

Comment:
I do actually prefer that, but I know @mvines has strong opinions about "utils"...

Reply:
I generally agree about _unqualified_ "utils" modules, since they just get full of stuff that shouldn't live together. With a qualifier, I'm a bit more lenient given there's some direction as to what should be there

Reply:
I'll push it, and see if he bites :)

Reply:
I heard that

Reply:
lemme counter with `solana-rpc-client-nonce-utils`. It's
1. Scoped by `solana-rpc-client`. Nice.
2. `-nonce`. Clearly something to do with nonces. Hmm, ok yeah, hey I'm trying to do a nonce thing.
3.  `-utils`. jfc, what does that mean!

Reply:
Nice, I like the `solana-rpc-client` scope bit, and the rest seems comparable to what we've got now.
Does that work for you @t-nelson ?

Reply:
:shipit:

Reply:
Done!